src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -14,6 +14,13 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        int countTotal = 0;
+        int count1,count2,count3,count4 = 0 ;
+        count1 = (a) ? 1 : 0;
```

Comment:
u still used ternary operators
refactor it according to checklist 
https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_01.html